dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -2,7 +2,7 @@
 
 # Try to read telemetry and certs config from ConfigDB.
 # Use default value if no valid config exists
-TELEMETRY_VARS=$(sonic-cfggen -d -t telemetry_vars.j2)
+TELEMETRY_VARS=$(sonic-cfggen -d -t /usr/bin/telemetry_vars.j2)
```

Comment:
why this j2 in the /usr/bin directory?

Reply:
ACK. moved it to templates dir. Will wait the image to test with.